internals/secrethub/run.go

```diff
@@ -498,6 +498,32 @@ func parseEnv(raw string) ([]envvar, error) {
 	return res, nil
 }
 
+const (
+	doubleQuoteChar = '\u0022'
```

Comment:
why not just `"` and `\'`? Those would be easier to read.

Reply:
Chose to use the unicode character to be explicit about which kind of quote we mean. There are quite a lot of different quotation marks in Unicode and we just mean the basic ones, so that's why I wanted to be explicit about it. The name also describes it already so I feel it's pretty clear as is. 

See: https://www.cl.cam.ac.uk/~mgk25/ucs/quotes.html

Reply:
How is this more explicit than setting a const to the rune itself?

```Go
const doubleQuoteChar = '"'
```
---
`'"' == '\u0022'`, see [this playground](https://goplay.space/#f8-DolKY9MD)